ndb/src/google/cloud/ndb/query.py

```diff
@@ -154,7 +154,7 @@ class Node:
     """Base class for filter expression tree nodes.
 
     Tree nodes are considered immutable, even though they can contain
-    Parameter instances, which are not.  In particular, two identical
+    Parameter instances, which are not. In particular, two identical
```

Comment:
It seems the original prose author uses two spaces after punctuation. I am being pedantic / wrong I suppose.

Reply:
FWIW, two spaces after period / colon / bang was "correct" in typewriter days:  it made things clearer for monspaced fonts, where we (ironically?) now spend all our time as programmers.